### PR TITLE
feat(csharp): add playground for all APIs

### PR DESCRIPTION
**Pull request by morganleroi**
## 🧭 What and Why

🎟 JIRA Ticket:

### Changes included:

- List changes

### CODE REVIEW

**Comment by shortcuts** [comment]
makes me think we could generate playgrounds... and just never push changes we made on them

cc @millotp I'm sure you'll want to implement that right away

**Comment by morganleroi** [comment]
It's done @millotp  !

File: playground/csharp/Playground/appsettings.example.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "AppID": "XXX",
+  "AdminApiKey": "XXXXXXXXXXXXXX",
```
Comment (millotp) [inline playground/csharp/Playground/appsettings.example.json:3]: you can't use the `.env` ?

**Review by millotp — COMMENTED** [review]
nice !

File: playground/csharp/Playground/appsettings.example.json (hunk 1 above)
Comment (morganleroi) [reply playground/csharp/Playground/appsettings.example.json:3]: `.env` is not the standard way to pass secret in dotnet. I can switch to `.env` file if you want. Dotnet dev are aware of the `appsetting` feature.

Comment (millotp) [reply playground/csharp/Playground/appsettings.example.json:3]: we are not Dotnet devs ahah, we don't want specific things for specific language, everything should use the same thing. Maybe you can leave both and default to `appsettings` if there is no .env ??

Comment (morganleroi) [reply playground/csharp/Playground/appsettings.example.json:3]: ok, let me add .env instead. Indeed in the head, the playground was mainly for our customers. But maybe not.

Comment (morganleroi) [reply playground/csharp/Playground/appsettings.example.json]: done

**Review by millotp — COMMENTED** [review]
Thanks for the change ! last comment my bad

File: playground/csharp/Playground/Utils/Configuration.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
+using dotenv.net;
+
+namespace Algolia.Utils;
+
+public static class Config
+{
+  public static Configuration Load()
+  {
+    DotEnv.Load(options: new DotEnvOptions(ignoreExceptions: false, envFilePaths: new[] { "./Playground/.env"}));
```
[the remark this answers is not in the document]
Comment (morganleroi) [reply playground/csharp/Playground/Utils/Configuration.cs]: ok done. For now, I still need other env var so I use both .env files.

[the remark this answers is not in the document]
Comment (millotp) [reply playground/csharp/Playground/Utils/Configuration.cs]: you could add them to the `.env.example` :)

**Review by millotp — APPROVED** [review]
gg !

**Pull request merged**